igc_lib.py

```diff
@@ -71,7 +73,148 @@ def _rawtime_float_to_hms(timef):
 
     return hms((time/3600), (time%3600)/60, time%60)
 
-
+class Turnpoint:
+    """ single turnpoint in a task.
+    Attributes: 
+        lat: a float, latitude in degrees 
+        lon: a float, longitude in degrees
+        radius: radius of cylinder or line in km
+        kind: type of turnpoint; "start_exit", "start_enter", "cylinder", "End_of_speed_section", "goal_cylinder", "goal_line"
```

Comment:
In general try to keep the lines under 80 characters. This line (and some other in the file) can easily be broken down.